src/blocks/Biometrics/utils/auth.android.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+/*
+  NOTE: react_native_touch_id_android needs a couple changes to the /android/build.gradle
+  and /android/app/src/main/java/com/rocknative/MainApplication.java file. If either of these
+  get changed, bad things may happen. Just so you know :)
+*/
+
+import auth from "react-native-touch-id-android";
```

Comment:
I believe you can write this like so:

```js
export {
  isSensorAvailable as isSupported,
  requestTouch as authenticate,
} from "react-native-touch-id-android"

Then you wouldn't export a default but the single functions. Since we don't need the singleton, this would help enforce only using the methods